social_core/backends/google.py

```diff
@@ -12,32 +12,25 @@ class BaseGoogleAuth(object):
     def get_user_id(self, details, response):
         """Use google email as unique id"""
         if self.setting('USE_UNIQUE_USER_ID', False):
-            return response['id']
+            if 'sub' in response:
```

Comment:
*EAFP*; I would suggest to use try catch, eg:
```python
try:
    return response['sub']
except KeyError:
    return response['id']
```

Reply:
I agree. I'd normally do that. But I followed the convention used in this module.